web/pgadmin/dashboard/templates/dashboard/sql/default/pgd_raft_status.sql

```diff
@@ -0,0 +1,14 @@
+with raft_status as (
+    select *
+    from json_to_record((select * from bdr.get_raft_status())) as rs(
+        instance_id oid, server_id oid, state text, leader_type text, leader oid,
+        voting bool, voted_for_type text, voted_for_id oid
+    )
+)
+select n.node_name, n.node_group_name, rs.*,
+    (select node_name from bdr.node_summary where node_id = rs.leader) as leader_name,
+    (select node_name from bdr.node_summary where node_id = rs.voted_for_id) as voting_for
+from bdr.node_summary n
+    left join raft_status rs
+    on n.node_id = rs.server_id
+order by n.node_seq_id;
```

Comment:
Keywords (select, from, where, left join, on, order by, as) should be in the capital for consistency.